src/components/breadcrumb-bar/utils/get-docs-breadcrumbs.ts

```diff
@@ -63,17 +63,22 @@ function getPathMatchedNode(navNodes, pathString) {
   if (matches.length > 1) {
     if (IS_DEV) {
       console.warn(
-        `Ambiguous breadcrumb path: Found ${matches.length} matches for "${pathString}". Please ensure there is exactly one node or index-less category with the path "${pathString}" in the provided navData.`
+        `Ambiguous breadcrumb path under "${basePath}": Found ${matches.length} matches for "${pathString}". Please ensure there is exactly one node or index-less category with the path "${pathString}" in the provided navData.`
       )
     }
     return matches[0]
   }
   // Otherwise, we have zero matches, which would mean a breadcrumb with missing parts.
-  // This would feel broken to a site visitor, so should be a blocking
-  // issue we need to fix, so we throw an error.
-  throw new Error(
-    `Missing breadcrumb path: Found zero matches for "${pathString}". Please ensure there is a node (or index-less category) with the path "${pathString}" in the provided navData.`
-  )
+  // We have no nav-data to render a nice "title" for the breadcrumb bar, and
+  // there isn't a matched path to link to, but we can still render an
+  // unlinked breadcrumb item using the "pathString" as the title
+  if (IS_DEV) {
+    console.warn(
+      `Missing breadcrumb path under "${basePath}": Found zero matches for "${pathString}". Please ensure there is a node (or index-less category) with the path "${pathString}" in the provided navData.`
+    )
+  }
+  const lastPathPart = pathString.split('/').pop()
+  return { title: lastPathPart }
 }
```

Comment:
Just leaving a comment for future reference, since we talked about it off this PR:

Sometime after we've onboarded content, and identified issues that need fixing, and fixed those issues, it would be nice to convert checks like this to:

1. Throw errors instead of warnings (so we can start enforcing content rules)
2. Make sure errors are thrown in all environments (is a more thorough check, warnings can probably still be dev only!)
3. Only throw errors when the latest version is being viewed, otherwise log a warning to the console (means we don't have to update every single version of content that conflicts with new/updated content rules)